MetabolicAtlas/3d-network-viewer
# chore: update three js version

**Related issue(s) and PR(s)**  

This PR closes #41 and #40.

**Type of change**  

 
**List of changes made**  

- Update `three` version.
- A few changes to make sure the project builds.

**Testing**  

1. Install newer versions of dependencies: `npm install`.
2. Verify that the development runs: `npm start` and that the network loads as expected in the browser. Please also verify that there are no console errors.
3. Verify that the project can build without any problems: `npm run build`.

**Further comments**  

`src/CSS2DRenderer.js` was removed since a newer version of it is available from the installed dependencies. 

**Definition of Done checklist**
